src/routes/safe/components/Apps/communicator.ts

```diff
@@ -66,8 +69,8 @@ class AppCommunicator {
           this.send(response, msg.data.id)
         }
       } catch (err) {
-        console.log({ err })
         this.send(err.message, msg.data.id, true)
+        logError(ErrorCodes._601, err.message)
```

Comment:
601 is a balances fetch error.
In this case you want a new error code.

Reply:
which one? it's fetching balances too 🤔 does it have to explicitly say this was inside a safe app?

Reply:
Isn't it the generic incoming message handler?

Reply:
🤦 right

Reply:
Please check it again, I'm not sure if the way I did it is sufficient as to me it seems there are two errors: handling safe apps request (more generic one) and fetching safe balances

Reply:
It would be good do separate them, yes.
We could
* throw a custom error from `fetchTokenCurrenciesBalances` (or a wrapper if you don't want to touch the original function)
* check the error code in the incoming message handler
* If it's the fetch error, log that, if not, log a generic error

Reply:
Sorry, I don't understand it. Let's say a safe app tries to fetch balances and the server responds with 429 Too Many Requests. What errors would you expect to be logged? What errors would you expect to be sent to Sentry?

In my opinion, it would be something like `${safeAppName} Error handling `getSafeBalances` request: 429 Too Many Requests` 

If I understood you right, what you suggest is to throw a custom error in the API function on the wrapper. How do we know that it occurred because of a call from the safe app? May be not enough info for debugging.